src/blue.ts

```diff
@@ -127,6 +127,14 @@ export function blueProxyFactory(env: MembraneBroker) {
         if (!isUndefined(normalizedRedDescriptor)) {
             const blueDesc = getBlueDescriptor(normalizedRedDescriptor);
             ReflectDefineProperty(shadowTarget, key, blueDesc);
+            // this cover the case in which the descriptor is not configurable, but writable
+            // while the line above will fail to set the descriptor, but setting the value
+            // directly into the shadowTarget using Reflect.set, which copy the value.
+            // This is faster than trying to compare the two descriptors to decide if we
+            // use defineProperty or Reflect.set()
```

Comment:
Comment cleanup
> // This covers the case in which the descriptor is non-configurable, but writable.
  // In that case the line above will fail to set the descriptor, but setting the value
   // directly into the shadowTarget using Reflect.set() will succeed.
  // This is faster than trying to compare the two descriptors to decide if we
  // use Reflect.defineProperty() or Reflect.set().

Reply:
Other than a comment cleanup it's good to go :shipit: